include/storage/io.hpp

```diff
@@ -104,6 +104,13 @@ class FileReader
         }
     }
 
+    void ReadInto(std::string &target)
+    {
+        auto len = ReadElementCount32();
```

Comment:
Depends how large the strings will be. 

```c++
numeric_limits<uint8_t>::max(); // -> 255
numeric_limits<uint16_t>::max(); // -> 65535
numeric_limits<uint32_t>::max(); // -> 4294967295
```

OSM guarantees for a max of 256*4 bytes per string, so a `uint16_t` would be fine here. Other strings not coming from osm might need more.